spec/omniauth/login_dot_gov/authorization_spec.rb

```diff
@@ -1,5 +1,6 @@
 describe OmniAuth::LoginDotGov::Authorization do
-  let(:client) { MockClient.new }
+  let(:aal) { nil }
+  let(:client) { MockClient.new({aal: aal}) }
```

Comment:
👀 this is not JS, the curly braces are optional

```suggestion
  let(:client) { MockClient.new(aal: aal) }
```